### DISCUSSION TITLE
Missing characters in Greek alphabet menu

### DISCUSSION THREAD

**Discussion by Donatien-Alphonse** [wxMaxima-developers/wxmaxima#1604] [Q&A] [answered]
Hello Friends,

Half of the Greek alphabet is missing from its menu in version 20.12.02 (appimage form) under Linux Mint 19.3. See the following image for full details.

![bad](https://user-images.githubusercontent.com/93812232/140598769-61070f22-43b3-45ad-be8e-ecbb1443a9ae.jpg)

Yet, in version 19.11.0, the entire alphabet shows up in the menu properly. See the next image.

![good](https://user-images.githubusercontent.com/93812232/140598773-dd9274b3-6531-4e1b-8c90-b3b65cf396d1.jpg)

Is this a Linux Mint incompatibility or a problem with wxMaxima? And any suggestions how I can overcome it?

Many thanks,

Donatien-Alphonse

**Comment by gunterkoenigsmann** [comment, marked as answer]
That was caused by a feature request from the maxima devs: an user used Greek an latin characters that looked exactly the same and got confused why the terms don't cancel each other out.  The discussion spanned to similarly-looking characters, as well.

For all users that can handle the whole Greek alphabet I have added a right-click menu to the Greek characters sidebar that enables them again

**Reply by Donatien-Alphonse** [reply]
Thank you so much for your quick response, Gunter. I had not noticed that feature. It works like a charm, and is a great option to accommodate all users.

I really do appreciate your work on this project, and kind willingness to help others out.

Best wishes,

Donatien-Alphonse
